packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -63,14 +64,62 @@ export class PostgresIndexApi implements DatabaseIndexApi {
   }
 
   async indexModels(models: Array<IndexModelArgs>): Promise<void> {
+    if (models.length === 0) return
+
     await initMidTables(this.dbConnection, models, this.logger)
     await this.verifyTables(models)
+    // FIXME: populate the updated_by field properly when auth is implemented
+    await this.dbConnection(INDEXED_MODEL_CONFIG_TABLE_NAME)
```

Comment:
please make a helper function for this so that this function is easier to read

Reply:
also note my comment on https://github.com/ceramicnetwork/js-ceramic/pull/2449 about whether we even want this functionality at all

Reply:
> whether we even want this functionality at all

Just to make sure I understand, @stbrody , which functionality were you referring to :) ?

Reply:
I meant the poor man's audit log of maintaining the `updated_by` column.